mkdocs/docs/HPC/teaching_training.md

```diff
@@ -0,0 +1,108 @@
+# Teaching and training
+
+The HPC infrastructure can be used for teaching and training pruposes, and
+we provide support with getting you organised.
+
+We assume that the course requirements are such that the interactive cluster
+can be used. If these are insufficient, you will need to request a reservation.
+
+[^TODO]: insert some links to the interactvie cluster?
```

Comment:
TODO to fix :)

Reply:
Should be https://docs.hpc.ugent.be/interactive_debug/